### PR TITLE
fix(button_height): add attribute in secondary and tertiary factory

### PR DESCRIPTION
**Pull request by LaurentMag**
Added missing attribute in others button factory

### CODE REVIEW

**Review by profotoce59 — APPROVED** [review]

File: flutter/lib/button.dart [hunk 1]
```diff
@@ -21,6 +21,7 @@ class Button extends StatefulWidget {
       text: text,
       onPressed: onPressed,
       color: kSecondaryColor,
+      buttonHeight: null,
```
Comment (PiotrFLEURY) [inline flutter/lib/button.dart:24]:
```dart
factory Button.secondary({
    Key? key,
    required String text,
    VoidCallback? onPressed,
    double? buttonHeight,
  }) {
    return Button(
      key: key,
      text: text,
      onPressed: onPressed,
      color: kSecondaryColor,
      buttonHeight: buttonHeight,
```

File: flutter/lib/button.dart [hunk 2]
```diff
@@ -35,6 +36,7 @@ class Button extends StatefulWidget {
       onPressed: onPressed,
       color: kTertiaryColor,
       fontColor: kSecondaryTextColor,
+      buttonHeight: null,
```
Comment (PiotrFLEURY) [inline flutter/lib/button.dart:39]:
```dart
factory Button.tertiary({
    Key? key,
    required String text,
    VoidCallback? onPressed,
    double? buttonHeight,
  }) {
    return Button(
      key: key,
      text: text,
      onPressed: onPressed,
      color: kTertiaryColor,
      fontColor: kSecondaryTextColor,
      buttonHeight: buttonHeight,
```

**Review by PiotrFLEURY — CHANGES_REQUESTED** [review]

File: flutter/lib/button.dart (hunk 1 above)
Comment (LaurentMag) [reply flutter/lib/button.dart]: fixed according to the suggestion for .secondary and tertiary

File: flutter/lib/button.dart (hunk 2 above)
Comment (LaurentMag) [reply flutter/lib/button.dart]: fixed according to the suggestion for .secondary and tertiary

**Review by PiotrFLEURY — APPROVED** [review]

**Pull request merged**